geotrek/infrastructure/models.py

```diff
@@ -148,7 +148,13 @@ def type_display(self):
 
     @property
     def cities_display(self):
-        return [str(c) for c in self.cities] if hasattr(self, 'cities') else []
+        all_cities_name = []
+        if hasattr(self, 'cities'):
```

Comment:
With Django you could do :
```suggestion
        if hasattr(self, 'cities'):
            all_cities_name = list(self.cities.objects.values_list('name', flat=True))
```
It think it's better to use SQL queryset to retrieve only the name from db